src/Umbraco.Cms.Integrations.Automation.Zapier/Migrations/ZapierSubscriptionHookTable.cs

```diff
@@ -34,44 +34,41 @@ public override void Migrate()
             Logger.LogDebug(MigrationLoggingMessage);
             
 #else
-            Logger.Debug<ZapContentConfigTable>(MigrationLoggingMessage);
+            Logger.Debug<ZapierSubscriptionHookTable>(MigrationLoggingMessage);
 #endif
 
-            if (TableExists(Constants.ZapContentConfigTable) == false)
+            if (TableExists(Constants.ZapierSubscriptionHookTable) == false)
             {
-                Create.Table<ZapContentConfig>().Do();
+                Create.Table<ZapierSubscriptionHook>().Do();
             }
             else
             {
 #if NETCOREAPP
                 Logger.LogDebug(DbTableExistsMessage);
 #else
-                Logger.Debug<ZapContentConfigTable>(DbTableExistsMessage);
+                Logger.Debug<ZapierSubscriptionHookTable>(DbTableExistsMessage);
 #endif
             }
         }
 
 
 
-        [TableName(Constants.ZapContentConfigTable)]
+        [TableName(Constants.ZapierSubscriptionHookTable)]
         [PrimaryKey("Id", AutoIncrement = true)]
         [ExplicitColumns]
-        public class ZapContentConfig
+        public class ZapierSubscriptionHook
         {
             [PrimaryKeyColumn(AutoIncrement = true, IdentitySeed = 1)]
             [Column("Id")]
             public int Id { get; set; }
 
-            [Column("ContentTypeName")]
-            [Index(IndexTypes.UniqueNonClustered, Name = "IX_ZapContentConfig_ContentTypeName")]
-            public string ContentTypeName { get; set; }
+            [Column("ContentTypeAlias")]
```

Comment:
Just thinking about how we'd extend this for other triggers, such as form submissions.  Would we create a new table?  Or reuse this table?

If the latter - and I think that makes sense - we would need to have this field more generically named, so it could store either a content type alias, or a form Guid, as appropriate.  So perhaps `EntityIdentifier`?